src/serac/infrastructure/cli.cpp

```diff
@@ -30,6 +30,8 @@ std::unordered_map<std::string, std::string> defineAndParse(int argc, char* argv
                "Writes Sphinx documentation for input file, then exits");
   std::string output_directory;
   app.add_option("-o, --output-directory", output_directory, "Directory to put outputted files");
+  bool enable_paraview{false};
+  app.add_flag("-p, --paraview", enable_paraview, "Enable paraview output");
```

Comment:
Please document this in the following file: `src/docs/sphinx/user_guide/command_line_options.rst`